arcgis-ios-sdk-samples/Edit data/Create and save KML file/CreateAndSaveKMLViewController.swift

```diff
@@ -0,0 +1,270 @@
+// Copyright © 2020 Esri.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import UIKit
+import ArcGIS
+
+class CreateAndSaveKMLViewController: UIViewController {
+    // Set the map.
+    @IBOutlet var mapView: AGSMapView! {
+        didSet {
+            mapView.map = AGSMap(basemap: .darkGrayCanvasVector())
+            resetKML()
+            let sketchEditor = AGSSketchEditor()
+            mapView.sketchEditor = sketchEditor
+        }
+    }
+    
+    @IBOutlet var addButton: UIBarButtonItem!
+    @IBOutlet var sketchDoneButton: UIBarButtonItem!
+    @IBOutlet var toolbar: UIToolbar!
+    @IBOutlet var actionButtonItem: UIBarButtonItem!
+    @IBOutlet var resetButtonItem: UIBarButtonItem!
+    
+    // Prompt feature selection action sheet.
+    @IBAction func addFeature() {
+        let alertController = UIAlertController(title: "Select Feature", message: nil, preferredStyle: .actionSheet)
+        let pointAction = UIAlertAction(title: "Point", style: .default) { (_) in
+            self.addPoint()
+        }
+        alertController.addAction(pointAction)
+        let polylineAction = UIAlertAction(title: "Polyline", style: .default) { (_) in
+            self.addPolyline()
+        }
+        alertController.addAction(polylineAction)
+        let polygonAction = UIAlertAction(title: "Polygon", style: .default) { (_) in
+            self.addPolygon()
+        }
+        alertController.addAction(polygonAction)
+        
+        // Add "cancel" item.
+        let cancelAction = UIAlertAction(title: "Cancel", style: .cancel, handler: nil)
+        alertController.addAction(cancelAction)
+        
+        alertController.popoverPresentationController?.barButtonItem = addButton
+        present(alertController, animated: true, completion: nil)
```

Comment:
`completion` has a default value of `nil`.

```suggestion
        present(alertController, animated: true)
```